src/pages/layer-2.js

```diff
@@ -0,0 +1,739 @@
+        </p>
+        <TwoColumnContent>
+          <FlexContainer flexPercent="65">
+            <h3>Rollups</h3>
+            <p>
+              Rollups are currently the preferred layer 2 solution for scaling
+              Ethereum. By using rollups, users can{" "}
+              <Link to="https://l2fees.info/">
+                reduce gas fees by up to 100x
+              </Link>{" "}
+              when comparison to a layer 1 transaction.
+            </p>
+            <p>
+              Rollups bundle (or ’roll up’) hundreds of transactions into a
+              single transaction on layer 1. This allows the transaction fees to
+              be shared across everyone in the rollup. These bundled
+              transactions are posted to the layer 1 as validity, fraud, or
+              fault proofs. Rollup transactions get executed outside of layer 1
+              but the transaction data gets posted to layer 1. By posting
+              transaction data onto layer 1, rollups are able to inherit the
+              security of Ethereum.
+            </p>
+          </FlexContainer>
+          <FlexContainer flexPercent="35">
+            <GatsbyImage image={getImage(data.impact)} />
+          </FlexContainer>
+        </TwoColumnContent>
+        <TwoColumnContent>
+          {rollupCards.map(
+            ({ emoji, title, description, childSentence, childLink }) => (
+              <RollupCard>
+                {emoji && <Emoji size={3} text={emoji} />}
+                <h3>{title}</h3>
+                <p>{description}</p>
+                <p>
+                  <Link to={childLink}>{childSentence}</Link>
+                </p>
+              </RollupCard>
+            )
+          )}
+        </TwoColumnContent>
+        <InfoBanner>
+          <h2>Do your own research: Risks of layer 2</h2>
+          <p>
+            Because layer 2 chains inherit security from Ethereum, in an ideal
+            world, they are as safe as L1 Ethereum. However, many of the
+            <b>projects are still young and somewhat experimental</b>. After
+            years of R&D, many of the L2 technologies that will scale Ethereum
+            went live in 2021. This is not to say these L2s are not secure, only
+            that no layer 2 is as battle tested as Ethereum Mainnet. Always do
+            your own research and decide if you're comfortable with any risks
+            involved.
+          </p>
+          <p>
+            For more information on the technology, risks and trust assumptions
+            of layer 2s, we recommend checking out L2BEAT, which provides a
+            comprehensive risk assessment framework of each project.
+          </p>
+          <p>
+            <Link to="https://l2beat.com">L2BEAT</Link>
+          </p>
+        </InfoBanner>
+      </Content>
+
+      <Content id="use-layer-2">
+        <h2>Generalized layer 2s</h2>
+        <p>
+          Generalized layer 2s behave just like Ethereum — but cheaper. Anything
```

Comment:
Ah, I saw someone make a change request for this, meant to comment. No space [is the correct usage of an em dash.](https://www.thepunctuationguide.com/em-dash.html)

```suggestion
          Generalized layer 2s behave just like Ethereum—but cheaper. Anything
```